<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QG_CadToolBarInfo</class>
 <widget class="QWidget" name="QG_CadToolBarInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>65</width>
    <height>336</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>65</width>
    <height>336</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Info</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>1</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="2">
    <widget class="QToolButton" name="bBack">
     <property name="toolTip">
      <string>Back to main menu</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/back.png</normaloff>:/extui/back.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>65</width>
       <height>20</height>
      </size>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QToolButton" name="bDist">
     <property name="toolTip">
      <string>Distance (Point, Point)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/infodist.png</normaloff>:/extui/infodist.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="bDist2">
     <property name="toolTip">
      <string>Distance (Entity, Point)</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/infodist2.png</normaloff>:/extui/infodist2.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QToolButton" name="bAngle">
     <property name="toolTip">
      <string>Angle</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/infoangle.png</normaloff>:/extui/infoangle.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QToolButton" name="bTotalLength">
     <property name="toolTip">
      <string>Total length of selected entities</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/infototallength.png</normaloff>:/extui/infototallength.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QToolButton" name="bArea">
     <property name="toolTip">
      <string>Area of polygon</string>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="icon">
      <iconset resource="../../../res/extui/extui.qrc">
       <normaloff>:/extui/infoarea.png</normaloff>:/extui/infoarea.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../../../res/extui/extui.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>bDist</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>infoDist()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bBack</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>back()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bDist2</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>infoDist2()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bAngle</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>infoAngle()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bTotalLength</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>infoTotalLength()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>bArea</sender>
   <signal>clicked()</signal>
   <receiver>QG_CadToolBarInfo</receiver>
   <slot>infoArea()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
